frontend/src/containers/SearchProperty.js

```diff
@@ -66,7 +64,12 @@ const SearchProperty = (props) => {
 };
 
 function mapStateToProps(state) {
-  return {};
+  return {
+    searchResults: {
+    ...state.search.searchResults,
+    pagination: state.search.searchResults.pagination,
+    }
+  };
```

Comment:
This will only render this container when the search results change

```suggestion
  return {
    results : state.search.searchResults
  };
```

Reply:
I will change this for now, but the response from the API includes pagination properties as well, so would these not be needed for rendering the results table? I'll admit I'm not entirely clear on the details about how pagination is being handled but again this is something we should clarify on the next call if possible.